js/api-augment/index.ts

```diff
@@ -38,6 +62,14 @@ export const signedExtensions: ExtDef = {
     extrinsic: {},
     payload: {},
   },
+  StaleHashCheckExtension: {
+    extrinsic: {},
+    payload: {},
+  },
+  StorageWeightReclaim: {
+    extrinsic: {},
+    payload: {},
+  },
```

Comment:
These are the additional signed extensions, but they don't need anything extra. This removes the warning.